apis/overview/lifecycle.md

```diff
@@ -1,20 +1,20 @@
 <!-- Copyright (c) Bentley Systems, Incorporated. All rights reserved.            -->
 <!-- See LICENSE in the project root for license terms and full copyright notice. -->
 
-# API release lifecycle
+# API versioning and release lifecycle
 
 ## API versions
 
-All APIs are versioned and the version number is embedded in the **Accept** header. Sample request:
+Our team versions all APIs and embeds the version number in the **Accept** header as shown below.
```

Comment:
Maybe it's a matter of taste, but do we need to focus on "our team" here? The main point is to highlight the versioning, not what our team does

Reply:
I feel what you are saying. I was trying to follow the style guide spot on. It says to use active voice vs. passive voice.

Reply:
Changed to we/us instead of "our team" throughout.